pino/pino-tests.ts

```diff
@@ -1,11 +1,11 @@
-/// <reference path="../node/node.d.ts"/>   // only for require()
+/// <reference path="../node/index.d.ts"/>   // only for require()
```

Comment:
do not use `/// <reference path=".." />`, use `/// <reference types="node" />` instead